operators/go/gen_subm_operator.sh

```diff
@@ -114,6 +114,8 @@ function add_subm_routeagent_to_operator() {
   sed -i '/RouteagentSpec struct/a \ \ SubmarinerNamespace string `json:"submariner_namespace"`' $types_file
   sed -i '/RouteagentSpec struct/a \ \ SubmarinerClusterid string `json:"submariner_clusterid"`' $types_file
   sed -i '/RouteagentSpec struct/a \ \ SubmarinerDebug string `json:"submariner_debug"`' $types_file
+  sed -i '/RouteagentSpec struct/a \ \ SubmarinerClustercidr string `json:"submariner_clustercidr"`' $types_file
+  sed -i '/RouteagentSpec struct/a \ \ SubmarinerServicecidr string `json:"submariner_servicecidr"`' $types_file
```

Comment:
ok, but still we need to push a patch to fix the naming of fields before release.

submariner_servicecidr is incorrect, and should be submarinerServiceCidr

I hope I can push it soon (after fixing the ipsec one).